com.github.jerempw.gnonograms.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "com.github.jeremypw.gnonograms",
+    "runtime": "org.gnome.Platform",
```

Comment:
Any reason for not using freedesktop runtime/sdk ?

Reply:
I'll try it.  I thought you needed the Gnome one for Vala.